science/memesuite

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                memesuite
+version             5.1.1
+distname            meme-${version}
+categories          science biology
+platforms           darwin linux
+license             Noncommercial
+maintainers         {@CharlesEGrant uw.edu:cegrant} \
+                    uw.edu:wnoble gmail.com:tlawbailey \
+                    openmaintainer
+description         Motif analysis toolkit
+long_description    MEME-Suite is a collection of tools for the analysis of \
+                    motifs in DNA and proteins.
+homepage            http://meme-suite.org
+master_sites        http://meme-suite.org/meme-software/macports \
+                    http://alternate.meme-suite.org/meme-software/macports
+
+checksums           rmd160  4a0b181454d136782e2fe2a5d0bce2f8363be478 \
+                    sha256  5bae8c2fa649bcafd6bd49dfe024d2e1d577cc802f9de2ba9dba3af05191093e \
+                    size    46161174
+
+depends_lib         port:expat \
+                    port:gawk \
+                    port:ghostscript \
+                    port:gsed \
+                    port:perl5.28 \
+                    port:p5.28-file-which \
+                    port:p5.28-html-template \
+                    port:p5.28-html-tree \
+                    port:p5.28-json \
+                    port:p5.28-math-cdf \
+                    port:p5.28-xml-parser \
+                    port:p5.28-xml-simple \
+                    port:python37 \
+                    port:py37-gnureadline \
+                    port:zlib
+
+configure.args      --enable-build-libxml2 \
+                    --enable-build-libxslt \
```

Comment:
Does this mean it uses an internal version of these libraries pegged at a specific version? Or can the Macports versions be used as dependencies?

Reply:
It uses an internal version of the libxml2 and libxslt libraries. We found our users had too many versions of the libraries to keep up with (including users with incompatible versions of libelist and libxml2)